main.go

```diff
@@ -86,7 +87,7 @@ func main() {
 	app.Run(os.Args)
 }
 
-func router(handler peopleHandler) http.Handler {
+func router(handler people.PeopleHandler) http.Handler {
```

Comment:
Assuming these will be used when handler is ready, can you please then remove the unnecessary TLS unsecure config for the http client? https://github.com/Financial-Times/v1-people-transformer/pull/1/files#diff-7ddfb3e035b42cd70649cc33393fe32cL113 It was accidentally left in v1-orgs-transformer as well :)